packages/ui/src/components/va-button/VaButton.demo.vue

```diff
@@ -1,295 +0,0 @@
-<template>
-  <VbDemo>
-    <VbCard class="va-button-demo">
-      <table class="table table-bordered">
-        <tr>
-          <td>Types</td>
```

Comment:
We need story for Presets. It should include all presets for button: 

default
primary
secondary
plain
plainOpacity

Reply:
Moved some items here - https://github.com/epicmaxco/vuestic-ui/issues/3898. But that's not something we need to worry in context of this pr.